gmso/formats/networkx.py

```diff
@@ -0,0 +1,953 @@
+                            getattr(node, label).to("angstrom").round(2) * unyt.angstrom
+                        )
+                    )
+                else:
+                    node.label = node.label + "\nNone"
+            else:
+                try:
+                    node.label = node.label + "\n" + str(getattr(node, label))
+                except AttributeError:
+                    node.label = node.label + "\nNoneType"
+                if len(node.label) > 12:
+                    node.label = "".join([line + "\n" for line in node.label.split()])
+        labels[node] = node.label
+
+    return labels
+
+
+def show_parameter_values(topology, list_of_params, checkbox):
+    if checkbox:
+        try:
+            report_parameter_expression(topology, list_of_params[0])
+        except AttributeError:
+            print("There are no values for the parameter expression")
+    else:
+        # TODO: Should be able to remove this blank print statement so that deselecting the
+        # checkbox removes the listed parameters.
+        print("")
+
+    return
+
+
+def report_parameter_expression(topology, param):
+    # return nicely printed parameters for a given edge.
+    if len(param) == 4:
+        try:
+            for dihedral in list(topology.dihedrals):
+                if dihedral.connection_members == (
+                    param[0],
+                    param[1],
+                    param[2],
+                    param[3],
+                ):
+                    print(dihedral.dihedral_type.expression, "\n")
+                    print("{:<12} {:<15}".format("Parameter", "Value"))
+                    for k, v in dihedral.dihedral_type.parameters.items():
+                        print("{:<12} {:<15}".format(k, v))
+        except AttributeError:
+            print("Dihedral not typed")
+    elif len(param) == 3:
+        try:
+            for angle in list(topology.angles):
+                if angle.connection_members == (param[0], param[1], param[2]):
+                    print(angle.angle_type.expression, "\n")
+                    print("{:<12} {:<15}".format("Parameter", "Value"))
+                    for k, v in angle.angle_type.parameters.items():
+                        print("{:<12} {:<15}".format(k, v))
+        except AttributeError:
+            print("Angle not typed")
+    else:
+        raise ValueError(
+            "Parameters are not proper angles or dihedrals. Connection members are missing"
+        )
+
+    return
+
+
+def get_edges(networkx_graph, atom_name1, atom_name2):
+
+    # Create a list of the edges that have been selected corresponding to the selected atom_name1
+    # and atom_name2. If both are None, then select edges that are missing bond types.
+    list_of_edges = []
+    list_of_bonds = []
+    # Check for whether you want to plot bonds based on the atom types, or their node labels.
+    try:
+        [node.atom_type.name for node in networkx_graph.nodes]
+    except AttributeError:
```

Comment:
Two questions:
1. Why is this returning None?
2. Do you want to fail for even single `atom_type` missing?

Reply:
I think that is the intended behavior. Since the main use case I was looking at was for reporting missing objects object parameters, I wanted these to be immediately reported. However, I could see a use where you're still trying to check a bond even when an atomtype is missing. Which wouldn't be possible with this line of code. I think in this case, I'll write a line of code where it only checks for missing atomtypes if that's the actual bond, angle, or dihedral you're looking for. In that case, it won't be able to show it because their is no name for that site.

Reply:
Gonna delete this try statement and check to see what happens if one of the nodes is missing an atomtype.

Reply:
So now we just see a missing atomtype on the networkx plot for atomtypes, and when plotting bonds and angles, you can select a particular bond, but it is blank for that particular site.